core/templates/dev/head/components/summary_tile/exploration_summary_tile_directive.html

```diff
@@ -40,24 +40,36 @@ <h3 class="activity-title protractor-test-exp-summary-tile-title">
               <span ng-if="getAverageRating()">
                 <[getAverageRating() | number:1]>
               </span>
-              <span ng-if="!getAverageRating()" translate="I18N_LIBRARY_N/A">
+              <span ng-if="!getAverageRating()" translate="I18N_LIBRARY_N/A" class=".oppia-activity-summary-tile .inactive-summary-tile-attribute ">
```

Comment:
Nope, the class should be just "inactive-summary-tile-attribute". Did you test this manually? Please always do so before submitting anything.

.oppia-activity-summary-tile is in line 2, and stacking CSS classes like ".class1 .class2" means to select elements with class "class2" that come below an element of class "class1".